package.json

```diff
@@ -27,13 +27,16 @@
   },
   "homepage": "https://github.com/nullndr/Matador#readme",
   "dependencies": {
+    "@mantine/core": "^4.2.5",
```

Comment:
Please, let's keep a defined version of the packages that we need to use, so delete the ^

Reply:
Fixed with aea788fd608c09e92c72ed67abd13d339c72d3ae